prime-router/settings/STLTs/VT/vt-doh.yml

```diff
@@ -0,0 +1,194 @@
+        receivingApplicationName: "NBS^2.16.840.1.114222.4.1.185.1^ISO"
+        receivingApplicationOID: "2.16.840.1.114222.4.1.185.1"
+        receivingFacilityName: "VDH^2.16.840.1.114222.4.1.185^ISO"
+        receivingFacilityOID: "2.16.840.1.114222.4.1.185"
+        messageProfileId: null
+        replaceValue:
+          MSH-3-1: "CDC PRIME - Atlanta,"
+        replaceValueAwithB: null
+        reportingFacilityName: null
+        reportingFacilityId: null
+        reportingFacilityIdType: null
+        suppressQstForAoe: false
+        suppressHl7Fields: "ORC-4-2, ORC-4-3, ORC-4-4, OBX-18-1, OBX-18-2, OBX-18-3,\
+        \ OBX-18-4, OBX-15-3, PID-15-1, PID-15-2, PID-15-3, MSH-19-1, MSH-19-2, MSH-19-3"
+        suppressAoe: true
+        defaultAoeToUnknown: false
+        replaceUnicodeWithAscii: false
+        useBlankInsteadOfUnknown: null
+        truncateHDNamespaceIds: true
+        truncateHl7Fields: "OBX-23-1"
+        usePid14ForPatientEmail: true
+        convertTimestampToDateTime: null
+        cliaForOutOfStateTesting: null
+        cliaForSender: {}
+        phoneNumberFormatting: "STANDARD"
+        suppressNonNPI: true
+        processingModeCode: null
+        replaceDiiWithOid: null
+        applyOTCDefault: false
+        useOrderingFacilityName: "STANDARD"
+        valueSetOverrides: {}
+        nameFormat: "STANDARD"
+        receivingOrganization: null
+        convertPositiveDateTimeOffsetToNegative: false
+        stripInvalidCharsRegex: null
+        convertDateTimesToReceiverLocalTime: false
+        useHighPrecisionHeaderDateTimeFormat: false
+        type: "HL7"
+        truncationConfig:
+          truncateHDNamespaceIds: true
+          truncateHl7Fields:
+            - "OBX-23-1"
+          customLengthHl7Fields: {}
+      jurisdictionalFilter:
+        - "(Bundle.entry.resource.ofType(ServiceRequest)[0].requester.resolve().organization.resolve().address.state = 'VT') or (Bundle.entry.resource.ofType(Patient).address.state = 'VT')"
+      qualityFilter:
+        - "Bundle.identifier.value.exists()"
+        - "Bundle.entry.resource.ofType(Patient).name.family.exists()"
+        - "Bundle.entry.resource.ofType(Patient).name.given.count() > 0"
+        - "Bundle.entry.resource.ofType(Patient).birthDate.exists()"
+        - "Bundle.entry.resource.ofType(Specimen).where(type.empty()).count() = 0"
+        - "Bundle.entry.resource.ofType(Patient).where(address.line[0].empty() and address.postalCode.empty() and telecom.where(system = 'phone').value.empty() and telecom.where(system = 'email').value.empty()).count() = 0"
+        - "((Bundle.entry.resource.ofType(Specimen).collection.collectedPeriod.exists() or Bundle.entry.resource.ofType(Specimen).collection.collected.exists())
+           or (Bundle.entry.resource.ofType(ServiceRequest).occurrence.exists() or Bundle.entry.resource.ofType(Observation).effective.exists()))"
+      routingFilter: []
+      #        - "matches(test_result_status, F, C, P)"
```

Comment:
For "matches(test_result_status, F, C, P)", please use the following fhirpath filter below:

        # - "matches(test_result_status, F, C, P)"
        - "Bundle.entry.resource.ofType(DiagnosticReport).where(status in 'final'|'corrected'|'preliminary'|'amended').exists()"